core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -8,6 +8,8 @@
  * @license LGPL-3.0-or-later
  */
 
+use Contao\CoreBundle\EventListener\DataContainer\Page\AdjustAutoforwardSubpaletteListener;
```

Comment:
Since DCA files are combined when the Symfony cache is warmed up, you should avoid `use` statements.

Reply:
What kind of issues would there be? I always use `use` statements and never experienced any issues. And since there are no issues, why not use it to improve the readability of the code?

Reply:
AFAIK `use` statements and `namespace` blocks are correctly handled by our `PhpFileLoader` class, so this should be fine.

Reply:
Sorry to keep nagging about this, but according to PhpStorm, multiple `use` statements that refer to the same class do **not** work:

<img width="750" alt="" src="https://user-images.githubusercontent.com/1192057/85296283-70988700-b4a1-11ea-9d7a-f3282649d8ac.png">

Reply:
My IDE complains about it too in the combined cache file. However, there are no PHP compiler errors.

```php
namespace {
    use \Foo\Foo;
}
namespace {
    use \Foo\Foo;
}
```

seems to be valid syntax. And imho it makes sense too - there will be lots of duplicate use statements throughout a project, but they are all encapsulated within their own namespace. Whether they are separated by files or not does not really matter.

Only this leads to a compiler error:

```php
namespace {
    use \Foo\Foo;
    use \Foo\Foo;
}
```

Reply:
Which PHP versions did you check?

Reply:
7.3 before, and now also 5.6. No compiler error there either. Seems to work in PHP 5.3 too.

Reply:
Confirmed on 3v4l.org: https://3v4l.org/2FioM